website/static/install.sh

```diff
@@ -21,15 +22,16 @@ help() {
     # Display Help
     echo "Install script for Oh My Posh"
     echo
-    echo "Syntax: install.sh [-h|d|t]"
+    echo "Syntax: install.sh [-h] [-d <directory>] [-t <directory>] [-v <version>]"
```

Comment:
```suggestion
    echo "Syntax: install.sh [-h] [-d <dir>] [-t <dir>] [-v <ver>]"
```

I think using abbreviation argument placeholders is a better match for the options (flags) short forms.